core/modules/user/user.admin.inc

```diff
@@ -288,6 +288,94 @@ function user_admin_settings_submit($form, &$form_state) {
+  );
+  $form['ip']['flood_ip_limit'] = array(
+    '#type' => 'select',
+    '#title' => t('Attempted login limit'),
+    '#description' => t('The number of times a user with a particular IP address can attempt to log in within a time window.'),
+    '#options' => backdrop_map_assoc(array(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 75, 100, 125, 150, 200, 250, 500)),
+    '#default_value' => $flood_config->get('flood_ip_limit'),
+  );
+  $form['ip']['flood_ip_window'] = array(
+    '#type' => 'select',
+    '#title' => t('Attempted login time window'),
+    '#description' => t('The window of time within which a user with a particular IP address can attempt to log in.'),
+    '#options' => array(0 => t('None (disabled)')) + backdrop_map_assoc(array(60, 180, 300, 600, 900, 1800, 2700, 3600, 10800, 21600, 32400, 43200, 86400), 'format_interval'),
+    '#default_value' => $flood_config->get('flood_ip_window'),
+  );
+
+  $form['user'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('User'),
+  );
+  $form['user']['help'] = array(
+    '#type' => 'help',
+    '#markup' => t('Configure the limit and the time window for users attempting to login. That is, how many failed login attempts are allowed every X hours.'),
+  );
+  $form['user']['flood_user_limit'] = array(
+    '#type' => 'select',
+    '#title' => t('Attempted login limit'),
+    '#description' => t('The number of times a user with a particular username can attempt to log in within a time window.'),
+    '#options' => backdrop_map_assoc(array(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 75, 100, 125, 150, 200, 250, 500)),
+    '#default_value' => $flood_config->get('flood_user_limit'),
+  );
+  $form['user']['flood_user_window'] = array(
+    '#type' => 'select',
+    '#title' => t('Attempted login time window'),
+    '#description' => t('The window of time a user with a particular username can attempt to log in.'),
+    '#options' => array(0 => t('None (disabled)')) + backdrop_map_assoc(array(60, 180, 300, 600, 900, 1800, 2700, 3600, 10800, 21600, 32400, 43200, 86400), 'format_interval'),
+    '#default_value' => $flood_config->get('flood_user_window'),
+  );
+  $form['user']['flood_uid_only'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Use only the User ID to identify users'),
+    '#description' => t('The default identifier is a combination of User ID and IP address. This is less secure but more resistant to denial-of-service attacks that could lock out all users with public user names. When enabled, checking for multiple login attempts will be based on the User ID only. This means that it will check regardless of the IP address from which the user is attempting to login.'),
+    '#default_value' => $flood_config->get('flood_uid_only'),
+  );
+
+  $form['actions']['#type'] = 'actions';
```

Comment:
You can avoid this part, and the submit handler, by using https://api.backdropcms.org/api/backdrop/core%21modules%21system%21system.module/function/system_settings_form/1